flutter_vlc_player_platform_interface/lib/src/platform_interface/vlc_player_platform_interface.dart

```diff
@@ -34,7 +34,7 @@ abstract class VlcPlayerPlatform extends PlatformInterface {
 
   /// Returns a widget displaying the video.
   Widget buildView(PlatformViewCreatedCallback onPlatformViewCreated,
-      {bool virtualDisplay = true}) {
+      {bool virtualDisplay = true, bool useTexture = false}) {
```

Comment:
What do you think if we move `useTexture` to class field like _token?

Reply:
If we move useTexture to class field of VlcPlayerPlatform, it will be static, and changing it will cause a mismatch between 2 types of player. useTexture need to be stored each time we create a player so we can find the correct player in native side.